utils/messages.js

```diff
@@ -22,5 +22,8 @@ module.exports = {
     OTP_SENT_SUCCESS: 'Otp sent successfully',
     INVALID_OTP: 'Invalid otp',
     OTP_EXPIRE: 'Otp is expired',
+    ADD_SUCCESS: 'data added successfully',
+    DELETE_SUCCESS: 'deleted successfully',
```

Comment:
ADD_SUCCESS and DELETE_SUCCESS already exist please use it and not add duplicate